stable/nginx-ingress/values.yaml

```diff
@@ -8,6 +8,12 @@ controller:
     tag: "0.17.1"
     pullPolicy: IfNotPresent
 
+  configMap:
+    create: true
```

Comment:
do we need both `configMap.create` and `configMap.name` ?   I would usually expect that if somebody is setting the configMap name they're implying that the configMap already exists.   maybe it could be as simple as:

```
# Set configMapName to be something other than nil (`~`) if yo want to use an existing ConfigMap rather than creating a new one.
configMapName: ~
```

Reply:
I'm fine with making this change - that seems like a good approach to me as well.